addons/website_slides/static/src/js/activity.js

```diff
@@ -42,4 +42,178 @@ function applyInclude(Activity) {
 
 applyInclude(KanbanActivity);
 
-});
\ No newline at end of file
+});
+
+odoo.define('website_slides/static/src/js/slide_models.js', function (require) {
+    'use strict';
+
+    const {
+        registerClassPatchModel,
+        registerFieldPatchModel,
+        registerNewModel
+    } = require('mail/static/src/model/model_core.js');
+    const { attr, one2one } = require('mail/static/src/model/model_field.js');
+
+    registerClassPatchModel('mail.activity', 'website_slides/static/src/js/slide_models.js', {
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+
+        /**
+         * @override
+         */
+        convertData(data) {
+            const data2 = this._super(data);
+            if ('request_partner_id' in data && 'res_id' in data) {
+                if (!data.request_partner_id) {
+                    data2.slide = [['unlink-all']];
+                } else {
+                    data2.slide = [
+                        ['insert', {
+                            id: data.res_id,
+                            partnerId: data.request_partner_id[0],
+                        }],
+                    ];
```

Comment:
This doesn't look correct, how do we know this is specifically a slide request? Shouldn't we check activity type or something else? The server must send something to let us know this is indeed a side request.

See approval has specific condition on `if ('approver_id' in data && 'approver_status' in data) {`